pipeline/data/dataset_importer.py

```diff
@@ -28,6 +29,8 @@
 COMP_CMD = os.getenv("COMPRESSION_CMD", "pigz")
 COMP_EXT = os.getenv("ARTIFACT_EXT", "gz")
 
+random.seed(1111)
```

Comment:
Interesting question. Just to make it clear: we set random seeds for reproducibility so that if you rerun the training with the same code you get the same results. In general, it is a best practice to set a hardcoded random seed. This code is used for validation/evaluation. For training even if we have the same random seed, the modifiers will be applied to a completely different corpus, so I don't see any issues here. We can totally change it to 2222 or any other number though :)

Reply:
Replying to this a bit late, but I think doing `random.seed(args.dataset)` would ensure reproducible results, but also ensure that we get good random distributions across different runs, as the dataset names should be stable.